components/Player/NotesTable.tsx

```diff
@@ -27,18 +147,24 @@ const Note: React.FunctionComponent<{ note: Notes }> = ({
         </div>
       </div>
       <p className="self-center row-span-1 mb-10 text-sm pt-3 mb-10">
-        {content}
+        {note.content}
       </p>
+      {note.authorId === userId && <EditDeleteMenu note={note} />}
```

Comment:
We want Admin to be able to edit all and mentors to only be able to edit the ones they wrote. So maybe add a check if the session user is an Admin and if so they get access